index.bs

```diff
@@ -7512,7 +7584,7 @@ In effect, where <var ignore>x</var> is a Number value,
         1.  If |signedness| is "<code>unsigned</code>", then let |lowerBound| be 0.
         1.  Otherwise let |lowerBound| be −2<sup>53</sup> + 1.
 
-            Note: this ensures {{long long}} types
+            Note: this ensures {{int64}} types
```

Comment:
This note pertained to both unsigned long long and long long, which already wasn’t very clear as written, but with “int64,” it’s even less clear that it means both variants (the upper bound clips both). Seems like listing both int64 and uint64 explicitly would fix that (or, given it’s a note, it could probably be worded like “64-bit integer types”).